cinema/models.py

```diff
@@ -4,7 +4,33 @@
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField("Actor", related_name="actors")
```

Comment:
The `related_name` for the `actors` field should be something like `movies` instead of `actors`. This is because `related_name` is used to access the reverse relation from the `Actor` model to the `Movie` model. Using `actors` might cause confusion as it suggests a self-referential relationship.